continue-reading.css

```diff
@@ -0,0 +1,201 @@
+    width: 70px;
+    background-color: #f2f3f4;
+    color: #B5838D;
+    font-size: 10px; 
+    border-radius: 3px;
+    font-style: bold;
+}
+button:hover {
+    color:blanchedalmond;
+    background-color: cadetblue;
+}
+
+/*-----------------HEADER---------------------*/
+#header {
+    display: block;
+    text-align: center;
+}
+
+/*-----------------MAIN SECTION---------------------*/
+/*grid to hold two columns*/ 
+#main {
+    display: grid;
+    grid-template-columns: 2fr 1fr;
+    grid-template-rows: 1fr minmax(5em, auto);
+    color: aquamarine;
+    grid-gap: 1em;
+}
+
+/*left side more in depth text*/
+.container1 {
+    grid-column: 1;
+    color: blue;
+    display: flex;
+    flex-direction: column;
+    padding: 2px; 
+}
+.large-image {
+    object-fit: cover;
+    max-width: 100%;
+    max-height: 100%; 
+}
+
+
+/*right side w/ 3 blog posts, 2 other posts*/
+.container2{
+    grid-column:2;
+    color: red;
+    display: flexbox;
+    flex-direction: column; 
+    padding: 2px;
+    padding-top: 65px;
+
+}
+/*want this text to float around the images*/
+.float-text {
+    color: purple;
+}
+
+.small-image {
+    width : 100px;
+    float: left;
+}
+
+/*-------------Footer - Contact Info Section----------*/
+
+/*entire section is a 4x5 gride*/
+#footer {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    grid-template-rows: 1fr 1fr 1fr 1fr 1fr;
+    color: #B5838D;
+    grid-gap: 1em;
+    background-color: blanchedalmond;
+    padding: 25px;
+}
+
+.logo {
+    width: 20px;
+}
+
+input {
+    color: gray;
+}
+
+input:focus{
+    color:darkcyan;
+    background-color: pink;
+}
+
+#name {
+grid-column: 1;
+grid-row:1;
+width: 100%;
+height: 100%;
+}
+#email {
+grid-column: 2;
+grid-row: 1; 
+width: 100%;
+height: 100%;
+}
+#message {
+    grid-column: 1/3;
+    grid-row: 2/5; 
+    width: 100%;
+    height: 100%;
+}
+#sendbutton {
+    grid-column:1;
+    grid-row: 5;
+    width: 100%;
+    height: 100%;
+}
+
+.description {
+    grid-column: 3/5; 
+    grid-row: 1;
+}
+
+.address {
+    display:inline;
+}
+.addresscontainer {
+    grid-column: 3;
+    grid-row: 2/3;
+    color: yellowgreen;
+    display: flexbox;
+}
+
+.phonecontainer {
+    grid-column: 3;
+    grid-row: 4;
+    display: flexbox
+
+}
+.phone {
+    display: inline;
+
+}
+
+.emailaddresscontainer {
+    grid-column: 3;
+    grid-row: 5; 
+    display: flexbox; 
+}
+.emailaddress {
+    display:inline
+
+}
+
+.tweetcontainer {
+    grid-column: 4;
+    grid-row: 2;
+    display: flexbox; 
+
+}
+.twitter {
+    display:inline;
+}
+
+.igcontainer {
+    grid-column: 4;
+    grid-row: 3;
+    display: flexbox;
+}
+.instagram {
+    display: inline; 
+}
+
+.dribble-container {
+    grid-column: 4;
+    grid-row: 4;
+}
+.dribble {
+    display: inline; 
+}
+
+.fbcontainer {
+    grid-column: 4;
+    grid-row: 5;
+}
+.facebook { 
+    display: inline;
+}
+
+@media (min-width: 75em) {
+    body {
+    margin-left: 2in; 
+    margin-right: 2in;
+    }
```

Comment:
Inside of this media query, you can redefine your grid sizes and where your elements live to adjust the structure of your page to just be one vertical column